src/main/java/redis/clients/jedis/executors/ClusterCommandExecutor.java

```diff
@@ -34,6 +37,40 @@ public void close() {
     this.provider.close();
   }
 
+  @Override
+  public final <T> T broadcastCommand(CommandObject<T> commandObject) {
+    Map<String, ConnectionPool> connectionMap = provider.getConnectionMap();
+
+    boolean isErrored = false;
+    T reply = null;
+    JedisBroadcastException holder = new JedisBroadcastException();
+    for (Map.Entry<String, ConnectionPool> entry : connectionMap.entrySet()) {
+      HostAndPort node = HostAndPort.from(entry.getKey());
+      ConnectionPool pool = entry.getValue();
+      try (Connection connection = pool.getResource()) {
+        try {
+          T aReply = execute(connection, commandObject);
+          holder.addReply(node, aReply);
+          if (isErrored) { // already errored
+          } else if (reply == null) {
```

Comment:
nit: the non okay cases should be together

Reply:
@chayim how else?

There's only one non-okay case (the `else` block), so all non okay are already together. Once non okay, I'm ignoring everything by first `if`.

Reply:
The reploy.equals first (because it's ok), then the replyy == null

Reply:
It'll cause a NullPointerException during the first iteration.